snuba/settings_base.py

```diff
@@ -36,6 +36,7 @@
 # Query Recording Options
 RECORD_QUERIES = False
 QUERIES_TOPIC = "snuba-queries"
+QUERY_LOG_DATASET_SAMPLING_RATE = 1
```

Comment:
Should this be a runtime option?

Reply:
I added it as a setting because of two reasons:
1) I suspect this is going to become structured to set dataset specific sampling rates (then we can gauge the accuracy with the `hits` field
2) I think we are likely to keep it permanently set. This may not be a killswitch, so we can track it in version control.